spec.md

```diff
@@ -849,11 +862,12 @@ In the following example there are three bonds, one between the atoms with the i
 
 #### bondOrderList
 
-*Optional field* If it exists [bondAtomList](#bondatomlist) must also be present. However `bondAtomList` may exist without `bondOrderList`.
+*Optional field* If it exists [bondAtomList](#bondatomlist) must also be present.
 
 *Type*: [Binary](#types) data that decodes into an array of 8-bit signed integers.
 
-*Description*: Array of bond orders for bonds in `bondAtomList`. Must be values between 1 and 4, defining single, double, triple, and quadruple bonds.
+*Description*: Array of bond orders for bonds in `bondAtomList`. Must be values between 0 and 4, defining unknown, single, double, triple, and quadruple bonds.
```

Comment:
How would you feel about `-1` being the `unknown` value? We also use `-1` for undefined secondary structure.
Motivation: One day we might discuss adding zero-order bonds.